docs/joss_paper/paper.md

```diff
@@ -54,63 +54,67 @@ bibliography: paper.bib
+However, popular packages focusing on deep learning methods for medical imaging, such as NiftyNet [@gibson2018niftynet] and MONAI (https://monai.io/), do not yet support image registration, while other open-source projects implmenting specific published algorithms, such as the VoxelMorph [@balakrishnan2019voxelmorph], are usually not tested or provided for general research and education purposes.
 
-`DeepReg` provides command line tool options that implement basic functionalities including model training, prediction and image warping. Options and configurations being specified through the command line arguments and/or configuration files in yaml format. This design choice has been motivated by both research and education purposes orginally set out in the DeepReg project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for developping novel methodology, rapid utilising latest development and accessing quality research topics in this area.
+`DeepReg` is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algortihms [@hu2018label] have been re-implemented. In addition, combining unsupervised losses, such as those derived from image similarity measures, and weak supervision, such as overlap between corresponding anatomical and pathological regions, is at the core of this framework. In this paper, we first briefly describe the major algorithms in DeepReg. A summary of the implementation of these algorithms is then provided. The model input and output, with- and without labels, the data sampling methods and the six dataset loaders implementing these are also described. Finally, we demonstrate the versatility of the package by providing a wide range of clinical applications as _DeepReg Demos_ in the `demos` folder, which are also briefly discussed in this paper.
+
+`DeepReg` provides command-line tool options that implement basic functionalities including model training, prediction and image warping. The yaml files and command-line arguments are used as the basis of configuring the various options with DeepReg. This design choice has been motivated by both research and education purposes originally set out in the DeepReg project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development and accessing quality research topics in this area.
```

Comment:
I got a bit confused reading the last sentence. Maybe "rapid utilising latest development" could be changed to "rapidly utilising latest developments in the field" if that does not change the meaning of the phrase.